packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/normal-dialing.js

```diff
@@ -14,8 +14,7 @@ describe('sample', () => {
       const browserMccoy = browser.select('browserMccoy');
 
       before('reload browser', () => {
-        browser.select('browserSpock').refresh();
-        browser.select('browserMccoy').refresh();
+        browser.refresh();
```

Comment:
So do the individual browsers not need refreshed, or this refreshes all?

Reply:
correct
`browser` by itself can send actions to multiple browsers (like `refresh`, `url`, and `setValue`), you need to call `select()` when trying to `getValue`